.changeset/stale-zoos-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/vite-plugin": minor
+---
+
+No longer call `next` in server middleware.
```

Comment:
Nit: it would be great if (after an empty line) you could add a bit of details as to why we're no longer calling next